docs/examples/modules/1vm-2nics-2subnets-1vnet/main.bicep

```diff
@@ -0,0 +1,155 @@
+param virtualMachineSize string
+param adminUsername string
+param adminPassword string {
+  secure: true
+}
+param storageAccountType string
+param location string = resourceGroup().location
+
+var nic1Name = 'nic-1'
+var nic2Name = 'nic-2'
+var virtualNetworkName = 'virtualNetwork'
+var subnet1Name = 'subnet-1'
+var subnet2Name = 'subnet-2'
+var publicIPAddressName = 'publicIp'
+var diagStorageAccountName = concat('diags', uniqueString(resourceGroup().id))
+var networkSecurityGroupName = 'NSG'
+var networkSecurityGroupName2 = concat(subnet2Name, '-nsg')
+
+module vmMod './vm.bicep' = {
```

Comment:
How are we going to decide if something should be a module or not in examples? The more we do it, the more we will start deviating from quickstart. I'm not necessarily against it, but it's worth mentioning...

Reply:
I've put these files in a different path (`docs/examples/modules`) - quickstarts are currently in `101`, `201` etc. I want to keep things in-line with the quickstarts repo (which is why I've put these in a different dir), but I think it would also be useful for us to be able to check in our own examples which more specifically showcase Bicep abilities.